services/galley/test/integration/API/Teams.hs

```diff
@@ -62,7 +62,7 @@ tests s = testGroup "Teams API"
     , test s "add team conversation as partner (fail)" testAddTeamConvAsExternalPartner
     , test s "add managed conversation through public endpoint (fail)" testAddManagedConv
     , test s "add managed team conversation ignores given users" testAddTeamConvWithUsers
-    , test s "add team member to conversation without connection" testAddTeamMemberToConv
+    , test s "add team member to conversation without connection" (testAddTeamMemberToConv >> ensureQueueEmpty)
```

Comment:
```suggestion
    , test s "add team member to conversation without connection" testAddTeamMemberToConv
```

(this is already happening at the end of the test)

Reply:
> (this is already happening at the end of the test)

Is it? Where? 😬

Reply:
oops, i may have skipped to [this line](https://github.com/wireapp/wire-server/blob/e4b5e1d9050ca3496ffa4d8f0d35482fb4aa97c5/services/galley/test/integration/API/Teams.hs#L711), which is a different test case.

anyway, it should be in the test IMO.  :)

(not a big deal)